README.md

```diff
@@ -64,6 +64,63 @@ The specified path inside the installer (`/cnab/app/terraform/terraform.tfstate`
 
 Alternatively, state can be managed by a remote backend.  When doing so, each action step needs to supply the remote backend config via `backendConfig`.  In the step examples below, the configuration has key/value pairs according to the [Azurerm](https://www.terraform.io/docs/backends/types/azurerm.html) backend.
 
+
+## Terraform variables file
+
+If you have required variables with no default or are overwriting default values, they should be persisted for terraform apply and destroy. Enabling
+`createVarFile` during install along with a `vars` block will create a
+[`terraform.tfvars.json`](https://www.terraform.io/docs/language/values/variables.html#variable-definitions-tfvars-files)
+during install.  Adding file parameter and output will persist it for upgrade
+and uninstall.
+
+The parameters for the terraform are only required at install and persisted
```

Comment:
Maybe, `"The parameters for Terraform that are supplied at install are persisted for subsequent steps."`?  Just to indicate that if the bundle author chooses, a parameter could apply to other actions besides `install`, perhaps if they wish for them to be overrideable on `upgrade`, etc. (naturally, this would require a `vars` section in `upgrade` with those parameters).